### PR TITLE
Use list to represent columns on Expr

### PR DESCRIPTION
**Pull request by phofl**
cc @rjzamora

### CODE REVIEW

File: dask_expr/expr.py [hunk 1]
```diff
@@ -491,9 +491,9 @@ def _name(self):
     @property
     def columns(self):
```
Comment (rjzamora) [inline dask_expr/expr.py:492]:
```suggestion
    def columns(self) -> list:
```
Maybe we start being explicit about the expected type here? (Certainly not a blocker)

File: dask_expr/collection.py [hunk 2]
```diff
@@ -96,6 +96,10 @@ def _meta(self):
     def size(self):
         return new_collection(self.expr.size)
 
+    @property
+    def columns(self):
```
Comment (rjzamora) [inline dask_expr/collection.py:100]: My preference is to stick with exactly what you have here, but just a thought: Do we want to raise an error for `Series.columns` (to be consistent with pandas)?

**Review by rjzamora — APPROVED** [review]
Nice. Thanks @phofl!

File: dask_expr/expr.py (hunk 1 above)
Comment (phofl) [reply dask_expr/expr.py]: Sounds good to me

File: dask_expr/collection.py (hunk 2 above)
Comment (rjzamora) [reply dask_expr/collection.py:100]: >We can add this here, this wouldn't break any tests

Up to you. This PR is already a win "as is" in my mind.

Comment (phofl) [reply dask_expr/collection.py:100]: Then let's merge as is, the columns change isn't urgent

**Pull request merged**